ddmrp/models/stock_buffer.py

```diff
@@ -176,6 +176,16 @@ def action_view_purchase(self):
         result["domain"] = [("id", "in", purchase_ids.ids)]
         return result
 
+    def action_view_yearly_consumption(self):
+        action = self.env.ref("ddmrp.stock_move_year_consumption_action")
+        result = action.read()[0]
+        result["domain"] = [
```

Comment:
I would use `_past_moves_domain` to get this domain, I will merge and open a new pr to fix.